### ISSUE TITLE
Assistive technologies read landmark label twice

### ISSUE THREAD

**Issue by michalrusina** [coralproject/talk#4664]
**Expected behavior:**

When using assistive technologies (screenreaders), all tabbed controls read the word "Tabs" twice. Once because of `aria-role="tabs"` and second time because there is an `span` element  containing text "Tabs".

**Actual behavior:** 

"Tabs" should be read only once, only use `aria-role="tabs"` where appropriate.

**Related Issues:**

This antipattern occurs on all tabbed controls and possibly also on other components. 

**Versions:**

- NodeJS: -
- NPM: -
- MongoDB: -
- Redis: -
- Browser: Chrome 128.0.6613.85
- OS: MacOS

**Comment by losowsky** [comment]
Thanks - we'll add this to our backlog

**Comment by losowsky** [comment]
Closing as this was included in the latest release, 9.4.0
